examples/vello_editor/src/main.rs

```diff
@@ -87,7 +88,7 @@ impl ApplicationHandler for SimpleVelloApp<'_> {
         // Save the Window and Surface to a state variable
         self.state = RenderState::Active(ActiveRenderState { window, surface });
 
-        event_loop.set_control_flow(ControlFlow::Poll);
+        event_loop.set_control_flow(ControlFlow::Wait);
```

Comment:
Is this here intentionally? It's also part of #144.

Reply:
`Poll` is excessive in this demo, `Wait` is appropriate.